prep.sh

```diff
@@ -41,9 +41,10 @@ do
 done
 
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
-version=0.5.1-rc1066.9cfd30140d30 # TODO https://github.com/jenkinsci/plugin-compat-tester/pull/276
-pct=$HOME/.m2/repository/org/jenkins-ci/tests/plugins-compat-tester-cli/${version}/plugins-compat-tester-cli-${version}.jar
-[ -f $pct ] || $MVN dependency:get -Dartifact=org.jenkins-ci.tests:plugins-compat-tester-cli:${version}:jar -DremoteRepositories=https://repo.jenkins-ci.org/public/,https://repo.jenkins-ci.org/incrementals/ -Dtransitive=false
+version=0.5.1-SNAPSHOT # TODO https://github.com/jenkinsci/plugin-compat-tester/pull/344 (c9c684eab7e8de91650ad1c96ded6489c42910f1)
```

Comment:
Better to get https://github.com/jenkinsci/plugin-compat-tester/pull/344 to pass tests—by ignoring stuff if necessary.

Reply:
ha, my last attempt of just reverting the PCT change that affected bom didn't go so well as there were other changes. Unless there's a simpler fix I'd probably just revert every PR in the last year =/. 

pr-merge pain -.-, can't build an isolated PR

Not sure if I'll have the time to fix properly.

Reply:
Ack. PCT needs a lot of work.